### PR TITLE
trying just using roll

### PR DESCRIPTION
**Pull request by muschellij2**
This PR I'm much less confident about (I'm not a Python programmer)

The goals are:

1. Not as much duplication with `lpf_sample_data`.  Though we still have a duplicate in the rolled data, we do not hav `lpf_upsample_data`, `upsample_data`, and `np.roll(lpf_upsample_data, 1)` in memory at one time
2. Use `np.roll` twice instead of the for loop.  I believe this is the intended addition.  
   - Aside, I'm not sure this is the same operation as in the `extract_slow` operation as https://github.com/actigraph/agcounts/blob/main/agcounts/extract.py#L126, but it seems like there is some shifting of indices due to the lagged zeroes added.
3, Using `column_stack` to add the zeros versus making `lpf_upsample_data`.

### CODE REVIEW

File: agcounts/extract.py [hunk 1]
```diff
@@ -283,30 +283,31 @@ def _resample(
     # rejected. This is the reason why there is aliasing which
     # causes the "tail" on the epochs.
     if frequency == 30 or frequency == 60 or frequency == 90:
-        lpf_upsample_data = upsample_data
+        print("lpf_data not needed", flush = True)
     else:
```
Comment (rouzbeh) [inline agcounts/extract.py:287]:
```python
if frequency not in [30, 60, 90]:
    upsample_data = (a_fp * up_factor_fp) * (
    ...
```

File: agcounts/extract.py [hunk 2]
```diff
@@ -283,30 +283,31 @@ def _resample(
...
-
     # Then allocate memory and downsample by factor M. Downsampled
     # data is rounded to 3 decimal places before input into BPF.
     if frequency == 30:
         downsample_data = raw
         del raw
     else:
-        downsample_data = lpf_upsample_data[:, ::downsample_factor]
+        downsample_data = upsample_data[:, ::downsample_factor]
     gc.collect()
```
Comment (rouzbeh) [inline agcounts/extract.py:308]: This line can be removed

**Review by rouzbeh — CHANGES_REQUESTED** [review]

File: agcounts/extract.py (hunk 1 above)
Comment (muschellij2) [reply agcounts/extract.py]: Fixed.

File: agcounts/extract.py (hunk 2 above)
Comment (muschellij2) [reply agcounts/extract.py]: Moved.

**Pull request merged**